src/index.js

```diff
@@ -36,34 +31,30 @@ export default class DesignSystem {
     return this.get(z, this.ds.zIndex)
   }
 
-  fontSize(size, toPxl = false) {
-    let output
-    if (this.options.useModularScale) {
-      const value =
-        typeof size === 'number' ? size : this.get(size, this.ds.type.sizes)
-      output = ms(value, this.ds.type.modularscale)
-    } else {
-      output = this.get(size, this.ds.type.sizes)
-    }
+  fontSize(size) {
+    const output = `${this.get(size, this.ds.type.sizes)}`
```

Comment:
Do you need a template literal here? Doesn't `this.get(size, this.ds.type.sizes)` already return a string with the unit?

Reply:
@munkychop `this.get()` returns whatever a user enters, so it could be a string, number, boolean etc.